Eevee.Sleep.Bot/Handlers/InteractionHandler.cs

```diff
@@ -116,4 +117,19 @@ await interaction
                 throw new ArgumentException($"Unhandled modal ID: {modalId}");
         }
     }
+
+    private static async Task OnButtonClicked(SocketMessageComponent component) {
+        var info = ButtonInteractionInfoSerializer.Deserialize(component.Data.CustomId);
+
+        switch (info?.ButtonId) {
+            case ButtonId.RoleChanger:
+                await component.RespondAsync("Your request has been accepted!", ephemeral: true);
```

Comment:
Maybe more details on this, but we can revisit this once all changes has been done.